rules/@typescript-no-watch/examples/correct.ts

```diff
@@ -0,0 +1,7 @@
+const event = {
```

Comment:
Let's add some smoke cases to correct test. E.g. `sample({ ... })`, `forward({...}), etc.